pg-index-health-testing/src/main/java/io/github/mfvanek/pg/testing/PostgreSqlClusterWrapper.java

```diff
@@ -49,15 +49,28 @@ public final class PostgreSqlClusterWrapper implements AutoCloseable {
     private final JdbcDatabaseContainer<?> containerForStandBy;
     private final BasicDataSource dataSourceForPrimary;
     private final BasicDataSource dataSourceForStandBy;
+    private final String username;
+    private final String password;
 
-    public PostgreSqlClusterWrapper() {
-        this.aliases = new PostgreSqlClusterAliasHolder();
+    private PostgreSqlClusterWrapper(final Builder builder) {
```

Comment:
It's better to pass independent strings here insted of using Builder

Reply:
Is this for clarity?